image/gimp_brush.ksy

```diff
@@ -0,0 +1,38 @@
+meta:
+  id: gimp_brush
+  title: GIMP (GNU Image Manipulation Program) brush version 2 file
+  file-extension: gbr
+  license: CC0-1.0
+  endian: be
+doc-ref: https://gitlab.gnome.org/GNOME/gimp/-/raw/master/devel-docs/gbr.txt
```

Comment:
Please don't link to the `master` branch that changes all the time - all URLs that you provide should be permanent (if possible). The target document should not change under our hands, which is something you have to expect when you refer to the `master` branch. There's also no guarantee that the document will remain available on the same path - it may be renamed or moved (to another path in the same repository or another repository).

I think that the reference docs should reflect the version that was used at the time of writing the spec. If a new version of the format is released, which has a different format structure than the former version, one would just open the URL from `doc-ref` and they would be presented with essentially a totally different format that doesn't reflect the one implemented by the KSY spec.

If you go to https://gitlab.gnome.org/GNOME/gimp/-/blob/master/devel-docs/gbr.txt, you can press <kbd>Y</kbd> on the keyboard, which will change the URL in the address bar to the permanent URL, which you can copy. Alternatively, there is also the following button in the top right corner (which does the same thing as <kbd>Y</kbd>):

<table>
<tbody>
<tr>
<td>Permalink</td>
</tr>
</tbody>
</table>

Then if you want to link to the raw file as @KOLANICH suggested in https://github.com/kaitai-io/kaitai_struct_formats/pull/427#discussion_r586725441, just change `/blob` to `/raw` in the URL:

```diff
- https://gitlab.gnome.org/GNOME/gimp/-/blob/441631322be109da6489b2aad670bdba916315c0/devel-docs/gbr.txt
+ https://gitlab.gnome.org/GNOME/gimp/-/raw/441631322be109da6489b2aad670bdba916315c0/devel-docs/gbr.txt
```